diff --git a/qb/config.libs.sh b/qb/config.libs.sh
index a4a1406363..8bf4d609d7 100644
--- a/qb/config.libs.sh
+++ b/qb/config.libs.sh
@@ -86,8 +86,14 @@ if [ "$HAVE_PRESERVE_DYLIB" = "yes" ]; then
 fi
 
 if [ "$HAVE_NEON" = "yes" ]; then
-   add_define MAKEFILE NEON_CFLAGS '-mfpu=neon -marm'
-   add_define MAKEFILE NEON_ASFLAGS -mfpu=neon
+   cust_fpu="$(echo ${CFLAGS} | grep -o "mfpu\=.[^ ]*" | sed 's/mfpu=//')"
+   if [ "x$cust_fpu" != "x" ]; then
+      add_define MAKEFILE NEON_CFLAGS '-mfpu=$cust_fpu -marm'
+      add_define MAKEFILE NEON_ASFLAGS '-mfpu=$cust_fpu'
+   else
+     add_define MAKEFILE NEON_CFLAGS '-mfpu=neon -marm'
+     add_define MAKEFILE NEON_ASFLAGS -mfpu=neon
+   fi
 fi
 
 if [ "$HAVE_FLOATHARD" = "yes" ]; then
