diff -U 3 -H -d -r -N -- a/qb/qb.libs.sh b/qb/qb.libs.sh
--- a/qb/qb.libs.sh	2014-02-18 18:27:45.000000000 +0100
+++ b/qb/qb.libs.sh	2014-02-18 20:32:54.343257239 +0100
@@ -14,7 +14,7 @@
 { echo "$1=$2" >> "$MAKEFILE_DEFINES";}
 
 add_include_dirs()
-{	while [ "$1" ]; do INCLUDE_DIRS="$INCLUDE_DIRS -I$1"; shift; done;}
+{	while [ "$1" ]; do INCLUDE_DIRS="$INCLUDE_DIRS -I/opt/nvidia-cg-toolkit/include -I$1"; shift; done;}
 
 add_library_dirs()
 {	while [ "$1" ]; do LIBRARY_DIRS="$LIBRARY_DIRS -L$1"; shift; done;}
