diff -U 3 -H -d -r -N RetroArch/Makefile b/Makefile
--- RetroArch/Makefile	2015-03-06 14:48:09.758239968 +0100
+++ b/Makefile	2015-03-06 14:53:09.490921624 +0100
@@ -138,8 +138,8 @@
 	rm -f $(OBJDIR)/git_version.o
 	mkdir -p $(DESTDIR)$(PREFIX)/bin 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(GLOBAL_CONFIG_DIR) 2>/dev/null || /bin/true
-	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1 2>/dev/null || /bin/true
-	mkdir -p $(DESTDIR)$(PREFIX)/share/pixmaps 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)/usr/share/man/man1 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)/usr/share/pixmaps 2>/dev/null || /bin/true
 	install -m755 $(TARGET) $(DESTDIR)$(PREFIX)/bin 
 	install -m755 tools/cg2glsl.py $(DESTDIR)$(PREFIX)/bin/retroarch-cg2glsl
 	install -m755 $(JTARGET) $(DESTDIR)$(PREFIX)/bin
@@ -147,8 +147,8 @@
 	install -m644 docs/retroarch.1 $(DESTDIR)$(MAN_DIR)
 	install -m644 docs/retroarch-cg2glsl.1 $(DESTDIR)$(MAN_DIR)
 	install -m644 docs/retroarch-joyconfig.1 $(DESTDIR)$(MAN_DIR)
-	install -m644 media/retroarch.png $(DESTDIR)$(PREFIX)/share/pixmaps
-	install -m644 media/retroarch.svg $(DESTDIR)$(PREFIX)/share/pixmaps
+	install -m644 media/retroarch.png $(DESTDIR)/usr/share/pixmaps
+	install -m644 media/retroarch.svg $(DESTDIR)/usr/share/pixmaps
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/bin/retroarch
diff -U 3 -H -d -r -N RetroArch/qb/qb.params.sh b/qb/qb.params.sh
--- RetroArch/qb/qb.params.sh	2014-10-19 13:22:47.583165360 +0200
+++ b/qb/qb.params.sh	2014-10-19 13:26:40.457431706 +0200
@@ -39,7 +39,7 @@
 opt_exists() # $opt is returned if exists in OPTS
 {	opt=$(echo "$1" | tr '[a-z]' '[A-Z]')
 	for OPT in $OPTS; do [ "$opt" = "$OPT" ] && return; done
-	print_help; exit 1
+	print_help; echo "Unknown option $1"
 }
 
 parse_input() # Parse stuff :V
@@ -67,7 +67,7 @@
 				eval "$opt=\"$val\""
 			;;
 			-h|--help) print_help; exit 0;;
-			*) print_help; exit 1;;
+			*) echo "Unknown option $1";;
 		esac
 		shift
 	done
