diff -U 3 -H -d -r -N a/Makefile b/Makefile
--- a/Makefile	2015-08-21 13:48:34.242292436 +0200
+++ b/Makefile	2015-08-21 13:51:07.860358992 +0200
@@ -158,18 +158,18 @@
 	rm -f $(OBJDIR)/git_version.o
 	mkdir -p $(DESTDIR)$(PREFIX)/bin 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(GLOBAL_CONFIG_DIR) 2>/dev/null || /bin/true
-	mkdir -p $(DESTDIR)$(PREFIX)/share/applications 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)/usr/share/applications 2>/dev/null || /bin/true
 	mkdir -p $(DESTDIR)$(MAN_DIR) 2>/dev/null || /bin/true
-	mkdir -p $(DESTDIR)$(PREFIX)/share/pixmaps 2>/dev/null || /bin/true
+	mkdir -p $(DESTDIR)/usr/share/pixmaps 2>/dev/null || /bin/true
 	install -m755 $(TARGET) $(DESTDIR)$(PREFIX)/bin 
 	install -m755 tools/cg2glsl.py $(DESTDIR)$(PREFIX)/bin/retroarch-cg2glsl
 	install -m755 $(JTARGET) $(DESTDIR)$(PREFIX)/bin
 	install -m644 retroarch.cfg $(DESTDIR)$(GLOBAL_CONFIG_DIR)/retroarch.cfg
-	install -m644 retroarch.desktop $(DESTDIR)$(PREFIX)/share/applications
+	install -m644 retroarch.desktop $(DESTDIR)/usr/share/applications
 	install -m644 docs/retroarch.1 $(DESTDIR)$(MAN_DIR)
 	install -m644 docs/retroarch-cg2glsl.1 $(DESTDIR)$(MAN_DIR)
 	install -m644 docs/retroarch-joyconfig.1 $(DESTDIR)$(MAN_DIR)
-	install -m644 media/retroarch.svg $(DESTDIR)$(PREFIX)/share/pixmaps
+	install -m644 media/retroarch.svg $(DESTDIR)/usr/share/pixmaps
 
 uninstall:
 	rm -f $(DESTDIR)$(PREFIX)/bin/retroarch
diff -U 3 -H -d -r -N a/qb/qb.params.sh b/qb/qb.params.sh
--- a/qb/qb.params.sh	2015-06-24 15:05:01.770214331 +0200
+++ b/qb/qb.params.sh	2015-08-21 13:51:59.493049010 +0200
@@ -48,7 +48,7 @@
 opt_exists() # $opt is returned if exists in OPTS
 {	opt=$(echo "$1" | tr '[a-z]' '[A-Z]')
 	for OPT in $OPTS; do [ "$opt" = "$OPT" ] && return; done
-	print_help; exit 1
+	print_help; echo "Unknown option $1"
 }
 
 parse_input() # Parse stuff :V
@@ -76,7 +76,7 @@
 				eval "$opt=\"$val\""
 			;;
 			-h|--help) print_help; exit 0;;
-			*) print_help; exit 1;;
+			*) echo "Unknown option $1";;
 		esac
 		shift
 	done
