diff --git a/qb/config.libs.sh b/qb/config.libs.sh
index a4a1406363..f841df4bc4 100644
--- a/qb/config.libs.sh
+++ b/qb/config.libs.sh
@@ -481,7 +481,7 @@ else
    check_lib '' VULKAN -lvulkan vkCreateInstance
 fi
 
-check_pkgconf PYTHON 'python3 python3 python-3.7 python-3.6 python-3.5 python-3.4 python-3.3 python-3.2'
+check_pkgconf PYTHON python-%PYTHON_VER%
 
 if [ "$HAVE_MENU" != 'no' ]; then
    if [ "$HAVE_OPENGL" = 'no' ] && [ "$HAVE_OPENGLES" = 'no' ] && [ "$HAVE_VULKAN" = 'no' ]; then
