diff -Naur EmulationStation_orig/es-app/src/CollectionSystemManager.cpp EmulationStation/es-app/src/CollectionSystemManager.cpp
--- EmulationStation_orig/es-app/src/CollectionSystemManager.cpp	2019-03-08 09:48:19.856279458 +0100
+++ EmulationStation/es-app/src/CollectionSystemManager.cpp	2019-03-08 09:49:54.363196919 +0100
@@ -11,7 +11,7 @@
 #include "Settings.h"
 #include "SystemData.h"
 #include "ThemeData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <fstream>
 
 std::string myCollectionsName = "collections";
diff -Naur EmulationStation_orig/es-app/src/Gamelist.cpp EmulationStation/es-app/src/Gamelist.cpp
--- EmulationStation_orig/es-app/src/Gamelist.cpp	2019-03-08 09:48:19.856279458 +0100
+++ EmulationStation/es-app/src/Gamelist.cpp	2019-03-08 09:51:00.830067582 +0100
@@ -6,7 +6,7 @@
 #include "Log.h"
 #include "Settings.h"
 #include "SystemData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 FileData* findOrCreateFile(SystemData* system, const std::string& path, FileType type)
 {
diff -Naur EmulationStation_orig/es-app/src/MetaData.cpp EmulationStation/es-app/src/MetaData.cpp
--- EmulationStation_orig/es-app/src/MetaData.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-app/src/MetaData.cpp	2019-03-08 09:49:18.293095392 +0100
@@ -2,7 +2,7 @@
 
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 MetaDataDecl gameDecls[] = {
 	// key,         type,                   default,            statistic,  name in GuiMetaDataEd,  prompt in GuiMetaDataEd
diff -Naur EmulationStation_orig/es-app/src/scrapers/GamesDBJSONScraper.cpp EmulationStation/es-app/src/scrapers/GamesDBJSONScraper.cpp
--- EmulationStation_orig/es-app/src/scrapers/GamesDBJSONScraper.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-app/src/scrapers/GamesDBJSONScraper.cpp	2019-03-08 09:50:43.020010886 +0100
@@ -10,7 +10,7 @@
 #include "Settings.h"
 #include "SystemData.h"
 #include "utils/TimeUtil.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 /* When raspbian will get an up to date version of rapidjson we'll be
    able to have it throw in case of error with the following:
diff -Naur EmulationStation_orig/es-app/src/scrapers/ScreenScraper.cpp EmulationStation/es-app/src/scrapers/ScreenScraper.cpp
--- EmulationStation_orig/es-app/src/scrapers/ScreenScraper.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-app/src/scrapers/ScreenScraper.cpp	2019-03-08 09:50:36.236656340 +0100
@@ -7,7 +7,7 @@
 #include "PlatformId.h"
 #include "Settings.h"
 #include "SystemData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <cstring>
 
 using namespace PlatformIds;
diff -Naur EmulationStation_orig/es-app/src/SystemData.cpp EmulationStation/es-app/src/SystemData.cpp
--- EmulationStation_orig/es-app/src/SystemData.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-app/src/SystemData.cpp	2019-03-08 09:50:05.116561814 +0100
@@ -10,7 +10,7 @@
 #include "Settings.h"
 #include "ThemeData.h"
 #include "views/UIModeController.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <fstream>
 #ifdef WIN32
 #include <Windows.h>
diff -Naur EmulationStation_orig/es-core/src/InputConfig.cpp EmulationStation/es-core/src/InputConfig.cpp
--- EmulationStation_orig/es-core/src/InputConfig.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-core/src/InputConfig.cpp	2019-03-08 09:53:04.320496927 +0100
@@ -1,7 +1,7 @@
 #include "InputConfig.h"
 
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 //some util functions
 std::string inputTypeToString(InputType type)
diff -Naur EmulationStation_orig/es-core/src/InputManager.cpp EmulationStation/es-core/src/InputManager.cpp
--- EmulationStation_orig/es-core/src/InputManager.cpp	2019-03-08 09:48:19.859612800 +0100
+++ EmulationStation/es-core/src/InputManager.cpp	2019-03-08 09:52:57.807139500 +0100
@@ -5,7 +5,7 @@
 #include "Log.h"
 #include "platform.h"
 #include "Window.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <SDL.h>
 #include <iostream>
 #include <assert.h>
diff -Naur EmulationStation_orig/es-core/src/MameNames.cpp EmulationStation/es-core/src/MameNames.cpp
--- EmulationStation_orig/es-core/src/MameNames.cpp	2019-03-08 09:48:19.862946141 +0100
+++ EmulationStation/es-core/src/MameNames.cpp	2019-03-08 09:52:45.630428193 +0100
@@ -3,7 +3,7 @@
 #include "resources/ResourceManager.h"
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <string.h>
 
 MameNames* MameNames::sInstance = nullptr;
diff -Naur EmulationStation_orig/es-core/src/Settings.cpp EmulationStation/es-core/src/Settings.cpp
--- EmulationStation_orig/es-core/src/Settings.cpp	2019-03-08 09:48:19.862946141 +0100
+++ EmulationStation/es-core/src/Settings.cpp	2019-03-08 09:52:39.583739558 +0100
@@ -3,7 +3,7 @@
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
 #include "platform.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <algorithm>
 #include <vector>
 
diff -Naur EmulationStation_orig/es-core/src/ThemeData.cpp EmulationStation/es-core/src/ThemeData.cpp
--- EmulationStation_orig/es-core/src/ThemeData.cpp	2019-03-08 09:48:19.862946141 +0100
+++ EmulationStation/es-core/src/ThemeData.cpp	2019-03-08 09:52:33.823718754 +0100
@@ -6,7 +6,7 @@
 #include "Log.h"
 #include "platform.h"
 #include "Settings.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <algorithm>
 
 std::vector<std::string> ThemeData::sSupportedViews { { "system" }, { "basic" }, { "detailed" }, { "grid" }, { "video" } };
diff -Naur EmulationStation_orig/external/CMakeLists.txt EmulationStation/external/CMakeLists.txt
--- EmulationStation_orig/external/CMakeLists.txt	2019-03-08 09:48:19.862946141 +0100
+++ EmulationStation/external/CMakeLists.txt	2019-03-08 09:51:42.943540493 +0100
@@ -2,4 +2,3 @@
 # package managers are included with the project (in the 'external' folder)
 
 add_subdirectory("nanosvg")
-add_subdirectory("pugixml")
