diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/CollectionSystemManager.cpp EmulationStation/es-app/src/CollectionSystemManager.cpp
--- EmulationStation_orig/es-app/src/CollectionSystemManager.cpp	2018-05-17 20:47:31.433344221 +0200
+++ EmulationStation/es-app/src/CollectionSystemManager.cpp	2018-05-17 20:48:43.843633703 +0200
@@ -11,7 +11,7 @@
 #include "Settings.h"
 #include "SystemData.h"
 #include "ThemeData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <fstream>
 
 std::string myCollectionsName = "collections";
diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/Gamelist.cpp EmulationStation/es-app/src/Gamelist.cpp
--- EmulationStation_orig/es-app/src/Gamelist.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-app/src/Gamelist.cpp	2018-05-17 20:49:40.563867216 +0200
@@ -6,7 +6,7 @@
 #include "Log.h"
 #include "Settings.h"
 #include "SystemData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 FileData* findOrCreateFile(SystemData* system, const std::string& path, FileType type)
 {
diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/MetaData.cpp EmulationStation/es-app/src/MetaData.cpp
--- EmulationStation_orig/es-app/src/MetaData.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-app/src/MetaData.cpp	2018-05-17 20:48:30.013577631 +0200
@@ -2,7 +2,7 @@
 
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 MetaDataDecl gameDecls[] = {
 	// key,         type,                   default,            statistic,  name in GuiMetaDataEd,  prompt in GuiMetaDataEd
diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/scrapers/GamesDBScraper.cpp EmulationStation/es-app/src/scrapers/GamesDBScraper.cpp
--- EmulationStation_orig/es-app/src/scrapers/GamesDBScraper.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-app/src/scrapers/GamesDBScraper.cpp	2018-05-17 20:49:23.730464006 +0200
@@ -6,7 +6,7 @@
 #include "PlatformId.h"
 #include "Settings.h"
 #include "SystemData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 using namespace PlatformIds;
 const std::map<PlatformId, const char*> gamesdb_platformid_map {
diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/SystemData.cpp EmulationStation/es-app/src/SystemData.cpp
--- EmulationStation_orig/es-app/src/SystemData.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-app/src/SystemData.cpp	2018-05-17 20:49:11.380413043 +0200
@@ -10,7 +10,7 @@
 #include "Settings.h"
 #include "ThemeData.h"
 #include "views/UIModeController.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <fstream>
 #ifdef WIN32
 #include <Windows.h>
diff -U 3 -dHrN -- EmulationStation_orig/es-app/src/SystemScreenSaver.cpp EmulationStation/es-app/src/SystemScreenSaver.cpp
--- EmulationStation_orig/es-app/src/SystemScreenSaver.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-app/src/SystemScreenSaver.cpp	2018-05-17 20:49:50.947243884 +0200
@@ -14,7 +14,7 @@
 #include "Renderer.h"
 #include "Sound.h"
 #include "SystemData.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <unordered_map>
 #include <time.h>
 
diff -U 3 -dHrN -- EmulationStation_orig/es-core/src/InputConfig.cpp EmulationStation/es-core/src/InputConfig.cpp
--- EmulationStation_orig/es-core/src/InputConfig.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-core/src/InputConfig.cpp	2018-05-17 20:52:24.441236181 +0200
@@ -1,7 +1,7 @@
 #include "InputConfig.h"
 
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 
 //some util functions
 std::string inputTypeToString(InputType type)
diff -U 3 -dHrN -- EmulationStation_orig/es-core/src/InputManager.cpp EmulationStation/es-core/src/InputManager.cpp
--- EmulationStation_orig/es-core/src/InputManager.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-core/src/InputManager.cpp	2018-05-17 20:51:28.647659611 +0200
@@ -5,7 +5,7 @@
 #include "Log.h"
 #include "platform.h"
 #include "Window.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <SDL.h>
 #include <iostream>
 #include <assert.h>
diff -U 3 -dHrN -- EmulationStation_orig/es-core/src/MameNames.cpp EmulationStation/es-core/src/MameNames.cpp
--- EmulationStation_orig/es-core/src/MameNames.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-core/src/MameNames.cpp	2018-05-17 20:51:46.407736612 +0200
@@ -3,7 +3,7 @@
 #include "resources/ResourceManager.h"
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <string.h>
 
 MameNames* MameNames::sInstance = nullptr;
diff -U 3 -dHrN -- EmulationStation_orig/es-core/src/Settings.cpp EmulationStation/es-core/src/Settings.cpp
--- EmulationStation_orig/es-core/src/Settings.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-core/src/Settings.cpp	2018-05-17 20:52:09.414503624 +0200
@@ -3,7 +3,7 @@
 #include "utils/FileSystemUtil.h"
 #include "Log.h"
 #include "platform.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <algorithm>
 #include <vector>
 
diff -U 3 -dHrN -- EmulationStation_orig/es-core/src/ThemeData.cpp EmulationStation/es-core/src/ThemeData.cpp
--- EmulationStation_orig/es-core/src/ThemeData.cpp	2018-05-17 20:47:31.436677568 +0200
+++ EmulationStation/es-core/src/ThemeData.cpp	2018-05-17 20:51:59.864461891 +0200
@@ -6,7 +6,7 @@
 #include "Log.h"
 #include "platform.h"
 #include "Settings.h"
-#include <pugixml/src/pugixml.hpp>
+#include <pugixml.hpp>
 #include <algorithm>
 
 std::vector<std::string> ThemeData::sSupportedViews { { "system" }, { "basic" }, { "detailed" }, { "grid" }, { "video" } };
diff -U 3 -dHrN -- EmulationStation_orig/external/CMakeLists.txt EmulationStation/external/CMakeLists.txt
--- EmulationStation_orig/external/CMakeLists.txt	2018-05-17 20:47:31.440010914 +0200
+++ EmulationStation/external/CMakeLists.txt	2018-05-17 20:50:30.820745197 +0200
@@ -2,4 +2,3 @@
 # package managers are included with the project (in the 'external' folder)
 
 add_subdirectory("nanosvg")
-add_subdirectory("pugixml")
